docs/en/Running-Hive-with-Alluxio.md

```diff
@@ -46,8 +46,8 @@ Hive's Hadoop jobs will store its intermediate results in Alluxio. Please follow
 
 ### Add additional Alluxio site properties to Hive
 
-If there are any Alluxio site properties you want to specify for Hive, add those to `hive-site.xml`. For example,
-change `alluxio.user.file.writetype.default` from default `MUST_CACHE` to `CACHE_THROUGH`:
+If there are any Alluxio site properties you want to specify for Hive, add those to `core-site.xml` to Hadoop configuration
+ directory on each node. For example, change `alluxio.user.file.writetype.default` from default `MUST_CACHE` to `CACHE_THROUGH`:
```

Comment:
hive or hdfs doesn't load the alluxio site properties, but the alluxio client invoked by hive should be able to create a Property instance based on the site property file if it is placed in hive classpath. At least this is the expected behavior